### PR TITLE
Voice rooms prototype

### PR DESCRIPTION
**Pull request by robintown**
See https://github.com/matrix-org/matrix-react-sdk/pull/8084 for the full details.

Notes: none
(notes are in the react-sdk PR)

---
This PR currently has no changelog labels, so will not be included in changelogs.

Add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is plus `X-Breaking-Change` if it's a breaking change.

### CODE REVIEW

File: src/vector/jitsi/index.ts [hunk 1]
```diff
@@ -142,25 +143,61 @@ let skipOurWelcomeScreen = false;
...
+                    ack(ev);
+                    if (meetApi && await meetApi.isVideoMuted()) {
+                        meetApi.executeCommand('toggleVideo');
+                    }
                 },
             );
             widgetApi.on(`action:${ElementWidgetActions.StartLiveStream}`,
                 (ev: CustomEvent<IWidgetApiRequest>) => {
                     if (meetApi) {
+                        ack(ev);
```
Comment (turt2live) [inline src/vector/jitsi/index.ts:193]: the widget API is expected to handle errors in these sorts of cases - can we move the ack to the bottom of all the handlers?

**Review by turt2live — COMMENTED** [review]
otherwise looks good - thanks!

File: docs/labs.md [hunk 2]
```diff
@@ -196,3 +196,7 @@ Threading allows users to branch out a new conversation from the main timeline o
 Threads can be access by clicking their summary below the root event on the room timeline. Users can find a comprehensive list of threads by click the icon on the room header button.
 
 This feature might work in degraded mode if the homeserver a user is connected to does not advertise support for the unstable feature `org.matrix.msc3440`  when calling the `/versions` API endpoint.
+
+## Voice & video rooms (`feature_voice_rooms`)
```
Comment (turt2live) [inline docs/labs.md:200]: from the react-sdk side: should this be listed as under development?

Comment (robintown) [reply docs/labs.md:200]: Yes it should

**Review by turt2live — APPROVED** [review]
this side lgtm

**Comment by turt2live** [comment]
eslint be less happy though, presumably because of the js-sdk refactor.

File: src/vector/jitsi/index.ts (hunk 1 above)
Comment (robintown) [reply src/vector/jitsi/index.ts]: Actually, it looks like moving them below the `await` points in the mute handlers is causing the widget API to spit out (harmless) "Unrecognized action" errors. What's the right thing to do in that case?

Comment (robintown) [reply src/vector/jitsi/index.ts]: @turt2live ping in case you missed this

**Pull request merged**